client/src/components/views/Contact.js

```diff
@@ -2,6 +2,7 @@ import React, { useState } from "react";
 import styled from "@emotion/styled";
 import Axios from "axios";
 import Modal from "react-modal";
+import { Grid, Image } from "semantic-ui-react";
```

Comment:
nice - however, if this is a new module, we need to run npm install
(which is fine, I like new things)